### PR TITLE
Fix: export data status command should not write in name registry

### PR DESCRIPTION
**Pull request by priyanshi-yb**
https://yugabyte.atlassian.net/browse/DB-14187

### CODE REVIEW

File: yb-voyager/cmd/exportDataStatusCommand.go [hunk 1]
```diff
@@ -69,7 +69,9 @@ var exportDataStatusCmd = &cobra.Command{
 		}
 		useDebezium = msr.IsSnapshotExportedViaDebezium()
 
-
+		if msr.SourceDBConf == nil {
```
Comment (makalaaneesh) [inline yb-voyager/cmd/exportDataStatusCommand.go:72]: There is a small window after sourceDBConf is created but before export data actually starts exporting tables. If export-data-status is called in that window what happens? 
Just confirming that we also have a check later to print the same message if status files are not yet created, right?

File: yb-voyager/src/namereg/namereg.go [hunk 2]
```diff
@@ -25,6 +25,7 @@ const (
 	SOURCE_DB_IMPORTER_ROLE         = "source_db_importer"         // Fallback.
 	SOURCE_REPLICA_DB_IMPORTER_ROLE = "source_replica_db_importer" // Fall-forward.
 	SOURCE_DB_EXPORTER_ROLE         = "source_db_exporter"
+	SOURCE_DB_EXPORT_STATUS_ROLE    = "source_db_export_status"
```
Comment (makalaaneesh) [inline yb-voyager/src/namereg/namereg.go:28]: nit: keep it source_db_exporter_status just for consistency?

File: yb-voyager/cmd/exportDataStatusCommand.go (hunk 1 above)
Comment (priyanshi-yb) [reply yb-voyager/cmd/exportDataStatusCommand.go:72]: Yes yes, that is already there

File: yb-voyager/src/namereg/namereg.go [hunk 3]
```diff
@@ -25,6 +25,7 @@ const (
 	SOURCE_DB_IMPORTER_ROLE         = "source_db_importer"         // Fallback.
 	SOURCE_REPLICA_DB_IMPORTER_ROLE = "source_replica_db_importer" // Fall-forward.
 	SOURCE_DB_EXPORTER_ROLE         = "source_db_exporter"
+	SOURCE_DB_EXPORT_STATUS_ROLE    = "source_db_exporter_status"
```
Comment (makalaaneesh) [inline yb-voyager/src/namereg/namereg.go:28]: nit: if you decided to change the value, change the variable name also? 😄

**Review by makalaaneesh — APPROVED** [review]
LGTM

**Pull request merged**